DNN Platform/DotNetNuke.Web.Client/ClientResourceSettings.cs

```diff
@@ -218,6 +277,29 @@ private static string GetPortalSettingThroughReflection(string settingKey)
             return null;
         }
 
+        private static string GetPortalSettingThroughReflection(int? portalId, string settingKey)

```

Comment:
suggestion: don't duplicate code

Rather than duplicating the logic from `GetPortalSettingThroughReflection(String)`, please update that method to call this one with the `portalId` parameter

Ideally this same pattern would apply to the other methods, as well.